src/components/MultiSelect/index.tsx

```diff
@@ -40,6 +41,8 @@ const MultiSelect: React.FC<IProps> = props => {
     if (isOpen && !!openRef && event.target !== openRef) setIsOpen(false)
   }
 
+  const dropdownIcon = isOpen ? 'up' : 'down'
```

Comment:
just 'icon'. It's a multi select but you call it dropdown which is another different term.

Reply:
Done